scss/_stepped-process.scss

```diff
@@ -0,0 +1,139 @@
+//
+// Orange stepped process
+//
+.stepped-process {
+  ol {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    overflow: hidden;
+    counter-reset: step;
+  }
+}
+
+.stepped-process-item {
+  display: flex;
+  flex: 1;
+  padding: map-get($spacers, 2);
+  margin-right: $stepped-process-margin-r;
+  font-weight: $font-weight-bold;
+  text-align: center;
+  list-style: none;
+  counter-increment: step;
+  background-color: $black;
+}
+
+.current {
+  background-color: $primary;
+
+  .stepped-process-link {
+    &,
+    &:hover,
+    &:focus,
+    &:active {
+      color: $black;
+    }
+  }
+
+  ~ .stepped-process-item {
+    background-color: $gray-400;
+
+    .stepped-process-link {
+      &,
+      &:hover,
+      &:focus,
+      &:active {
+        color: $black;
+      }
+    }
+  }
+}
+
+.stepped-process-link {
+  flex: 1 0 1.25ch;
+  max-width: 1.25ch;
+  margin: auto;
+  overflow: hidden;
+  color: $white;
+  text-decoration: if($link-decoration == none, null, none);
+  white-space: nowrap;
+  outline-offset: map-get($spacers, 3);
+
+  &::before {
+    content: counter(step) "\A0";
+  }
+
+  &:hover,
+  &:focus,
+  &:active {
+    color: $white;
+  }
+
+  &:hover {
+    text-decoration: if($link-hover-decoration == underline, null, underline);
+  }
+
+  &:focus {
+    text-decoration: $link-decoration;
+    outline-offset: map-get($spacers, 1);
+  }
+}
+
+@include media-breakpoint-up(sm) {
+  .stepped-process-item {
+    max-width: 3.75rem;
```

Comment:
Hardcoded magical values like this, and most of values based on variables, should be defined as variables in `_variables.scss` for the component. We'll do this at some point.

Reply:
Yes I could have been more careful, because I took it from v4 as-is.